src/server/utils/watchLocalModules.js

```diff
@@ -17,54 +17,165 @@
 // This file is only used in development so importing devDeps is not an issue
 /* eslint "import/no-extraneous-dependencies": ["error", {"devDependencies": true}] */
 
-import fs from 'fs';
-import path from 'path';
-import chokidar from 'chokidar';
```

Comment:
if this is the only place chokidar is used (directly), can we remove it from package.json?